tests/x11/multi_users_dm.pm

```diff
@@ -75,16 +75,17 @@ sub run {
     # Make sure screen changed before calling handle_login function (for slow workers)
     wait_still_screen;
     handle_login($user, 1);
+    send_key 'esc'                        if match_has_tag('gnome-activities');
     handle_welcome_screen(timeout => 120) if (opensuse_welcome_applicable);
     assert_screen 'generic-desktop', 60;
     # verify correct user is logged in
     x11_start_program('xterm');
-    wait_still_screen;
+    assert_and_click('xterm-white-area');
```

Comment:
@punkioudi this needle is way too generic (if I see correctly, it's just a white square box), I suggest to change that to something that can be better differentiated or rather, more specific.

Reply:
I don't think this click should be necessary. IIUC, it's to get gnome out of activities mode, but that should already be done by this point.

Reply:
You are 100% correct, it is my bad, it is a leftover from previous workarounds.